app/helpers/validation_search.py

```diff
@@ -162,6 +163,14 @@ def bbox(self, value):
                     msg = "The third coordinate must be higher than the fourth"
                     logger.error(msg)
                     raise BadRequest(msg)
+            try:
+                # Python 2/3
```

Comment:
Why this comment ? We don't need to support python 2 anymore and the line 170 is only supported on python 3.6 and above.

Reply:
its a copy paste relict